using_openshift/port_forwarding.adoc

```diff
@@ -0,0 +1,110 @@
+= Port Forwarding
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+You can use the CLI to forward one or more local ports to a pod. This allows you to listen on a given or random port locally, and have data forwarded to and from given ports in the pod.
+
+== Basic Usage
+Support for port forwarding is build into
+link:../using_openshift/cli.html#common-cli-operations[the CLI]:
+
+****
+`$ osc port-forward -p _<Pod>_ _[<Local_Port>:]<Pod_Port>_ _[[<Local_Port>:]<Pod_Port> ...]_`
+****
+
+The CLI listens on each local port specified by the user, forwarding via the link:#protocol[protocol] described below.
+
+Ports may be specified using the following formats:
+
+`5000`::
+  The client listens on port 5000 locally and forwards to 5000 in the
+pod.
+`6000:5000`::
+  The client listens on port 6000 locally and forwards to 5000
+in the pod.
+`:5000` or `0:5000`::
+  The client selects a free local port and
+forwards to 5000 in the pod.
+
+== Examples
+For example, to listen on ports `5000` and `6000` locally and forward data to and from ports `5000` and `6000` in the pod, run:
+
```

Comment:
Yeah, wasn't sure about this originally either. I've just dropped the Examples heading now.